assets/components/contributionSelection/contributionSelection.jsx

```diff
@@ -42,24 +42,27 @@ type PropTypes = {|
   error: ContributionError,
   oneOffSingleOneTimeTestVariant: 'control' | 'single' | 'once' | 'oneTime' | 'notintest',
   usOneOffSingleOneTimeTestVariant: 'control' | 'single' | 'once' | 'oneOff' | 'notintest',
+  annualTestVariant: 'control' | 'annual' | 'notintest',
 |};
 
 
 // ----- Component ----- //
 
 function ContributionSelection(props: PropTypes) {
 
-  const modifierClass = getContributionTypeClassName(props.contributionType);
+  const modifierClassArray = [getContributionTypeClassName(props.contributionType)];
+  if (props.annualTestVariant === 'annual') { modifierClassArray.push('annual'); }
```

Comment:
Doesn't `getContributionTypeClassName(props.contributionType)` return the string `annual` already?

Reply:
This is used to display the styling for the test (buttons not radio etc.) and so needs to return 'annual' even when 'monthly' or 'one-off' are returned by `getContributionTypeClassName`. This would be clearer if I'd called the modifier `annual-test`.